utils_nlp/models/transformers/named_entity_recognition.py

```diff
@@ -77,7 +87,7 @@ def get_inputs(batch, device, model_name, train_mode=True):
                 Labels are only returned when train_mode is True.
         """
         batch = tuple(t.to(device) for t in batch)
-        if model_name.split("-")[0] in ["bert", "distilbert"]:
+        if model_name in list(TC_MODEL_CLASS):
```

Comment:
model_name is not optional in this function. The documentation needs to be changed.

Reply:
can do_lower_case be handled within the function, instead of being passed as an argument to avoid mismatch?